databricks/koalas/frame.py

```diff
@@ -7710,9 +7710,9 @@ def melt(self, id_vars=None, value_vars=None, var_name=None, value_name="value")
 
         Examples
         --------
-        >>> df = ks.DataFrame({'A': {0: 'a', 1: 'b', 2: 'c'},
-        ...                    'B': {0: 1, 1: 3, 2: 5},
-        ...                    'C': {0: 2, 1: 4, 2: 6}},
+        >>> df = ks.DataFrame({'A': {'0': 'a', '1': 'b', '2': 'c'},
+        ...                    'B': {'0': '1', '1': '3', '2': '5'},
+        ...                    'C': {'0': '2', '1': '4', '2': '6'}},
```

Comment:
Why changed here? Seems not related?

Reply:
It was because of `array(struct(...), ...)` function type coercion in Spark 2.3. Nested structs require the exactly same type in Spark 2.3. From Spark 2.4, seems they can coerce types properly. So I had to change it to strings to make it pass in Spark 2.3 too.

Reply:
Oops, yes. Seems I mistakenly changed it.